README.md

```diff
@@ -16,13 +16,13 @@ To use N-API in a native module:
   1. Add a dependency on this package to `package.json`:
 ```json
   "dependencies": {
-    "node-addon-api": "github:nodejs/node-addon-api",
+    "node-addon-api": "github:nodejs/node-api",
   }
 ```
 
   2. Reference this package's include directory and gyp file in `binding.gyp`:
 ```gyp
-  'include_dirs': ["<!(node -p \"require('node-addon-api').include\")"],
+  'include_dirs': ["<!@(node -p \"require('node-addon-api').include\")"],
```

Comment:
It indicates that the output of the command is to be used to fill out an array. In this case, it uses the double quotes I add in index.js to correctly split the output into array elements, stripping the quotes, so the paths are not misinterpreted.